flycheck.el

```diff
@@ -5501,14 +5501,26 @@ CHECKER."
     `(
       ,(flycheck-verification-result-new
         :label "executable"
-        :message (if executable (format "Found at %s" executable) "Not found")
+        :message
+        (if executable (format "Found at %s" executable)
+          (format
+           "Not found. %s"
+           (let ((candidate (flycheck-checker-executable checker)))
+             (if candidate (format "(Tried %s)" candidate)
+               "(No executable file was specified for this checker.)"))))
```

Comment:
I worry that this message will mislead users into thinking that they must specify an executable for Flycheck to work.  I'd be more comfortable with displaying that extra message only if an executable was already specified, but couldn't be found.

Reply:
If a user has not specified an executable, then flycheck will use the value of `(flycheck-checker-default-executable checker)`. So my understanding is that we do not expect a user to see this message; but if they do, they do indeed need to specify an executable because no default was provided.

Reply:
That doesn't sound right; what about the case where the user just hasn't installed the tool yet?

Reply:
Thanks! The ball is in my court here; I will think think this through more carefully in light of your review.